.github/workflows/yarn.yml

```diff
@@ -1,23 +1,30 @@
 # This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name: Node.js CI
+name: Yarn CI & CD
 
 on:
-  push:
-    branches: [ main ]
+  # runs on all pull requests against main
   pull_request:
     branches: [ main ]
+  # this is a reusable workflow that is called from the release workflow
+  workflow_call:
+    inputs:
+      release:
+        required: false
+        type: boolean
 
 jobs:
   build:
 
-    runs-on: ubuntu-latest
-
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
         node-version: [16.x]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    outputs:
+      # define the short-SHA here as an output of the build step
```

Comment:
Same, I would skip this comment as I don't think it carries more information that the code itself.

Reply:
Fixed in latest commit!